packages/core/src/routes-me/user.ts

```diff
@@ -141,23 +137,25 @@ export default function userRoutes<T extends AuthedMeRouter>(
       const { id: userId } = ctx.auth;
       const { password } = ctx.guard.body;
 
-      const { customData, isSuspended } = await findUserById(userId);
+      const {
+        passwordVerificationStatus: { verifiedAt = 0, sessionId: storedSessionId },
```

Comment:
just double check this will be updated by #3312

Reply:
Yes, it will. Just rebased and this is gone